### ISSUE TITLE
Support for openai/transcription

### ISSUE THREAD

**Issue by flebarbe** [genai-impact/ecologits#91]
### Description

Hi,

I'm looking at implementing [openai's transcription API](https://platform.openai.com/docs/api-reference/audio/createTranscription) in EcoLogits.

It seems doable: We do have the tokens in the returned data (see `segments` in the  [transcription object](https://platform.openai.com/docs/api-reference/audio/verbose-json-object)), so we have a metric to base our calculation on.

Is anyone already working on this ? Do you forsee any obstacle ?

Thanks !

**Comment by samuelrince** [comment]
Hello @flebarbe,

As we discussed IRL, we first need to have energy benchmarks on speech-to-text models before implementing it!
